client/src/components/CesiumViewer.vue

```diff
@@ -0,0 +1,57 @@
+/*#############################################################################
+# Copyright Kitware Inc. and Contributors
+# Distributed under the Apache License, 2.0 (apache.org/licenses/LICENSE-2.0)
+# See accompanying Copyright.txt and LICENSE files for details
+#############################################################################*/
+
+<template>
+  <div>
+    <vc-viewer @ready="ready">
+      <vc-primitive-tileset :url="url" @readyPromise="readyPromise" />
+    </vc-viewer>
+  </div>
+</template>
+
+<script>
+import { mapState } from "vuex";
+import { loadWorkingSetById, getTilesetFolderId } from "../utils/loadDataset";
+
+import { API_URL } from "../constants";
+
+export default {
+  asyncComputed: {
+    async url() {
+      const data = await loadWorkingSetById(this.selectedWorkingSetId);
+      if (data.output_folder_id) {
+        // get path of output folder using its id
+        const tilesetFolderId = await getTilesetFolderId(data.output_folder_id);
+
+        return `${API_URL}/resource/path/download/${tilesetFolderId}/tiler/tileset.json`;
+      }
+      return null;
+    },
+  },
+  computed: {
+    ...mapState({
+      selectedWorkingSetId: "selectedWorkingSetId",
+    }),
+  },
+  methods: {
+    ready(cesiumInstance) {
+      this.cesiumInstance = cesiumInstance;
+      const { Cesium, viewer } = cesiumInstance;
+    },
```

Comment:
I think it'd be good to declare `cesiumInstance` in the data as `null`, so that when it's set here, things are clear. Also, I think the second line can be removed:

```suggestion
    ready(cesiumInstance) {
      this.cesiumInstance = cesiumInstance;
    },
```